src/vs/workbench/contrib/chat/browser/chatResponseAccessibleView.ts

```diff
@@ -65,6 +64,19 @@ class ChatResponseAccessibleProvider extends Disposable implements IAccessibleVi
 		if (!responseContent && 'errorDetails' in item && item.errorDetails) {
 			responseContent = item.errorDetails.message;
 		}
+		if (isResponseVM(item)) {
+			const toolInvocation = item.response.value.find(item => item.kind === 'toolInvocation');
+			if (toolInvocation?.confirmationMessages) {
+				const title = toolInvocation.confirmationMessages.title;
+				const message = typeof toolInvocation.confirmationMessages.message === 'string' ? toolInvocation.confirmationMessages.message : toolInvocation.confirmationMessages.message.value;
+				const terminalCommand = toolInvocation.toolSpecificData && 'command' in toolInvocation.toolSpecificData ? toolInvocation.toolSpecificData.command : undefined;
```

Comment:
The 'input' type data could be useful to people too. It's just that it could be a huge chunk of JSON.

Also, fyi we will likely extend `toolSpecificData` in the future. You could set it up so that there's a TS error here if it's extended to make sure we don't forget to update this when a new type of data is added. Or add a comment to the `IPreparedToolInvocation.toolSpecificData`

Reply:
https://github.com/microsoft/vscode/pull/247062